airflow/hooks/__init__.py

```diff
@@ -15,8 +15,10 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-# fmt: off
+# fmt:, off
```

Comment:
```suggestion
# fmt: off
```

Reply:
(Does it even do anything?)

Reply:
Oh yeah They were small corrections I've fixed to make the PR's gree. 

Yeah. They actualy do  (and I think the comma does not really matter) - they prevent Black from re-formatting those files . The problem with such long names  of classes we have in the dict that (despite the max-line-length) it will make lines too long and they fail in flake. One of the very few cases where black is WRONG.